### PR TITLE
Fix Array module docstrings

### PR DESCRIPTION
**Pull request by DZakh**

### CODE REVIEW

File: test/ArrayTests.res [hunk 1]
```diff
@@ -4,6 +4,9 @@ let eq = (a, b) => a == b
 
 Test.run(__POS_OF__("make"), Array.make(~length=6, 7), eq, [7, 7, 7, 7, 7, 7])
 
+Test.run(__POS_OF__("getUnsafe - existing"), [0, 1, 2]->Array.getUnsafe(1), eq, 1)
+Test.run(__POS_OF__("getUnsafe - missing"), [0, 1, 2]->Array.getUnsafe(10), eq, %raw(`undefined`))
```
Comment (DZakh) [inline test/ArrayTests.res:8]: In practice it happened not to raise 😅

**Comment by zth** [comment]
So the existing fn is wrong...?

**Comment by DZakh** [comment]
@zth What about merging the PR?

**Comment by dsiu** [comment]
> If you're talking about `Array.getUnsafe`, I think it works as it should. What was wrong is the docstrings.

Yup. It works as it should just like JS but the doc was incorrect. Thanks for the fix.

**Pull request merged**